src/main/java/ca/intelliware/ihtsdo/mlds/domain/AbstractAuditingEntity.java

```diff
@@ -35,18 +32,18 @@ public abstract class AbstractAuditingEntity {
 
     @CreatedDate
     @NotNull
-    @Type(type = "org.jadira.usertype.dateandtime.joda.PersistentDateTime")
+//    @Type(type = "org.jadira.usertype.dateandtime.joda.PersistentDateTime")
     @Column(name = "created_date")
-    private DateTime createdDate = DateTime.now();
+    private Instant createdDate = Instant.now();
```

Comment:
@dmcgihtsdo @type annotation is not supported in SB3 version. so it has been removed and Upgraded to java time module from joda time module so because of this change DateTime is replaced with Instant.